src/main/java/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this configuration file for the task?

Reply:
I use3 it for testing in my db:) Removed it - thanks!